frontend/public/components/modals/create-namespace-modal.jsx

```diff
@@ -129,20 +129,63 @@ const CreateNamespaceModalWithTranslation = connect(
         return (
           <>
             <p>{nameFormat}</p>
-            {this.props.createProject ? <p>{createNamespace}</p> : null}
+            {createProject ? <p>{createNamespace}</p> : null}
           </>
         );
       };
+
+      const popoverDescriptionText = (
+        <>
+          <p>
+            <strong>{t('public~OpenShift Projects and Kubernetes Namespaces')}</strong>
+          </p>
+          <p>
+            {t(
+              'public~Projects are exposed as editable to end users while namespaces are not. Direct creation of a project is typically restricted to administrators, while end users should use the requestproject resource.',
+            )}
+          </p>
+          <p>
+            <a
+              className="co-external-link"
+              href="https://docs.openshift.com/online/pro/architecture/core_concepts/projects_and_users.html"
```

Comment:
Use `openshiftHelpBase` to build doc URLs. That will make sure the links go to the right product version of the doc.

Reply:
I couldn't find the link to the page [](https://docs.openshift.com/online/pro/architecture/core_concepts/projects_and_users.html with the )`openshiftHelpBase`.  Do you know how the get to it?

Reply:
@maudem If this link is specific to the OpenShift Online doc, we'd want to pick a different topic that's available in other versions of the doc, maybe https://docs.openshift.com/container-platform/4.8/applications/projects/working-with-projects.html